File: Largest_number.cpp
// C++ program to find the largest number in an array
// without using std::max_element
// Simplified Version 
#include <iostream>

using namespace std;

int main()
{
    // take input of array size
    int n;
    cin>>n;
    //take input of array
    int arr[n] ;
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }

    // Initialize the max_num with the first element of the array
    int max_num = arr[0];

    // Traverse the array to find the largest number
    for (int i = 1; i < n; i++) {
        if (arr[i] > max_num) {
            max_num = arr[i];
        }
    }

    // Output the result
    cout << "Largest number in the array is: " << max_num << endl;

    return 0;
}

File: largest_number.py
import java.util.Scanner;

public class LargestNumber {

    public static int findLargestNumber(int[] arr) {
        if (arr.length == 0) {
            return Integer.MIN_VALUE; // Return a minimum value if the array is empty
        }

        int largest = arr[0]; // Assume the first number is the largest

        for (int num : arr) {
            if (num > largest) {
                largest = num; // Update largest if current number is bigger
            }
        }

        return largest;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of elements: ");
        int n = scanner.nextInt();
        int[] numbers = new int[n];

        // Input elements
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + ": ");
            numbers[i] = scanner.nextInt();
        }

        // Finding the largest number
        int largestNumber = findLargestNumber(numbers);
        
        if (largestNumber != Integer.MIN_VALUE) {
            System.out.println("The largest number is: " + largestNumber);
        } else {
            System.out.println("The array is empty.");
        }

        scanner.close();
    }
}
